usaspending_api/awards/models/award.py

```diff
@@ -99,10 +99,20 @@ class Award(DataSourceTrackedModel):
         help_text="The total of the face_value_loan_guarantee from associated transactions",
     )
     awarding_agency = models.ForeignKey(
-        "references.Agency", related_name="+", null=True, help_text="The awarding agency for the award", db_index=True
+        "references.Agency",
+        on_delete=models.CASCADE,
```

Comment:
It is dangerous to cascade deletes here, (hopefully no significant delete scripts are using the ORM). Is setting `DO_NOTHING` a possibility? A SQL failure causing a transaction rollback would be more optimal since it would keep awards in the table.

Reply:
Setting DO_NOTHING is definitely possible. The only reason they are CASCADE here is to mimic current functionality.